ServiceProviderBot/appsettings.Development.json

```diff
@@ -16,5 +16,5 @@
   "ConnectionStrings": {
     "DbModel": "data source=(LocalDb)\\MSSQLLocalDB;initial catalog=OrganizationModel;integrated security=True;MultipleActiveResultSets=True;App=EntityFramework"
   },
-  "MapsKey": "<create personal key in azure. Will have one to share when Azure grant comes through>"
```

Comment:
We updated this with a key so that we can get map location services during development.